src/codegen/codegen_c_modif_visitor.cpp

```diff
@@ -0,0 +1,29 @@
+/*************************************************************************
+ * Copyright (C) 2018-2022 Blue Brain Project
+ *
+ * This file is part of NMODL distributed under the terms of the GNU
+ * Lesser General Public License. See top-level LICENSE file for details.
+ *************************************************************************/
+
+#include <iostream>
+#include <memory>
+#include <unordered_set>
+
+#include "ast/ast_decl.hpp"
+#include "ast/function_block.hpp"
+#include "ast/string.hpp"
+#include "ast/table_statement.hpp"
+#include "codegen/codegen_c_modif_visitor.hpp"
+#include "visitors/visitor_utils.hpp"
+
+namespace nmodl {
+using namespace ast;
+
+void CodegenCModifVisitor::visit_function_block(FunctionBlock& node) {
```

Comment:
`nitpick`: name could be `CodegenTransformVisitor` indicating all transformations/modifications related to code generation. And filename `codegen_transform_visitor.cpp/hpp`.